test/testlib/nuodb_admin_utilities.go

```diff
@@ -142,6 +143,17 @@ func StartAdminTemplate(t *testing.T, options *helm.Options, replicaCount int, n
 		}
 	}
 
+	// License is mandatory for running test with NuoDB 6.0
+	RunOnNuoDBVersionFromOptionCondition(t, options, ">=6.0.0", func(version *semver.Version) {
+		if os.Getenv("NUODB_LIMITED_LICENSE_CONTENT") != "" {
+            ApplyLicense(t, namespaceName, adminNames[0], LIMITED)
+		} else if os.Getenv("NUODB_LICENSE_CONTENT") != "" {
+            ApplyLicense(t, namespaceName, adminNames[0], ENTERPRISE)
+        } else {
```

Comment:
Nit: Could you please fix the code formatting?

Reply:
sure